components/timezone/src/ixdtf.rs

```diff
@@ -262,21 +298,42 @@ impl<'a> Intermediate<'a> {
             self.time.minute,
             self.time.second,
         )?;
```

Comment:
Should we now be checking for `is_z` and failing with `MismatchedTimeZoneFields` in `location_only`?

Reply:
I think so.